pkg/services/store/object/object.proto

```diff
@@ -3,51 +3,65 @@ package object;
 
 option go_package = "./;object";
 
-// The canonical object/document data -- this represents the raw bytes and storage level metadata
-message RawObject {
-  // Unique ID
-  string UID = 1;
+message GRN {
+  // the tenant/org id
+  int64 tenant_id = 1;
+
+  // mabybe "namespace"?  valid values include
+  // * entity
+  // * drive
+  // * service/xyz
+  string scope = 2;
 
   // Identify the object kind.  This kind will be used to apply a schema to the body and
   // will trigger additional indexing behavior.
-  string kind = 2;
+  string kind = 3;
+
+  // Unique ID
```

Comment:
lets note down what other properties do we need for global uniqueness? is `tenantId` + `UID` enough? or do we need  all four: `tenantId` + `scope` + `kind` + `uid`?

Reply:
yes all 4